src/config/test.config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "apiBase": "https://raha-test.appspot.com/api",
+  "apiBase": "https://raha-test.appspot.com",
```

Comment:
Did this not work for some reason?

Reply:
unfortunately `url.resolve(base, path)` stripped the `/api` from the base url. Probs a way to not do that, I didn't bother looking for this fix.

Reply:
```
> require("url").resolve("http://google.com/api", "meow")
'http://google.com/meow'
> require("url").resolve("http://google.com/api/", "meow")
'http://google.com/api/meow'
```